### PR TITLE
fix: Update Sketch Colors to match design specs

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we update the sketch colors and updated the color picker to match the design specs. In addition to that we did a little bit of cleanup and provided a bit of documentation for certain classes.

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios/Wire-iOS/Sources/UserInterface/SketchColorPickerController.swift [hunk 1]
```diff
@@ -188,7 +198,7 @@ extension SketchColorPickerController: UICollectionViewDataSource, UICollectionV
         let color = sketchColors[indexPath.row]
         if selectedColor == color {
             // The color is already selected -> Change the brush size for this color
-            let brushWidth = bumpBrushWidth(for: color)
+            let brushWidth = bumpBrushWidth(for: color.color)
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/UserInterface/SketchColorPickerController.swift]: `color.color` looks like a duplicate name. Let's find better names

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/SketchColorPickerController.swift]: @KaterinaWire Done!

File: wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift [hunk 2]
```diff
@@ -191,6 +191,27 @@ public enum SemanticColors {
         static let backgroundPickUp = UIColor(light: Asset.Colors.green500Light, dark: Asset.Colors.green500Dark)
         static let textUnderlineEnabledDefault = UIColor(light: Asset.Colors.black, dark: Asset.Colors.white)
     }
+
+    public enum SketchColors {
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift]: For localizable strings, you use "drawing" and here "sketch". Maybe it's worse to use the same names. What do you think?

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift]: Done!

File: wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift [hunk 3]
```diff
@@ -191,6 +191,27 @@ public enum SemanticColors {
         static let backgroundPickUp = UIColor(light: Asset.Colors.green500Light, dark: Asset.Colors.green500Dark)
         static let textUnderlineEnabledDefault = UIColor(light: Asset.Colors.black, dark: Asset.Colors.white)
     }
+
+    public enum DrawingColors {
+        static let blackColor = UIColor(light: Asset.Colors.black, dark: Asset.Colors.black)
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift]:
```suggestion
        static let black = UIColor(light: Asset.Colors.black, dark: Asset.Colors.black)
```

File: wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift [hunk 4]
```diff
@@ -191,6 +191,27 @@ public enum SemanticColors {
...
+        static let whiteColor = UIColor(light: Asset.Colors.white, dark: Asset.Colors.white)
+        static let blueColor = UIColor(light: Asset.Colors.blue500Light, dark: Asset.Colors.blue500Light)
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift]: the enum already contains the word "color"

File: wire-ios/Wire-iOS/Sources/UserInterface/SketchColorPickerController.swift [hunk 5]
```diff
@@ -111,7 +121,7 @@ final class SketchColorPickerController: UIViewController {
 
         var colorToBrushWidthMapper: [UIColor: CGFloat] = [:]
         for color in sketchColors {
-            colorToBrushWidthMapper[color] = brushWidth
+            colorToBrushWidthMapper[color.color] = brushWidth
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/UserInterface/SketchColorPickerController.swift]: `color.color`

**Review by KaterinaWire — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/UserInterface/SketchColorPickerController.swift [hunk 6]
```diff
@@ -43,7 +43,12 @@ final class SketchColorPickerController: UIViewController {
             resetColorToBrushWidthMapper()
 
             colorsCollectionView.reloadData()
-            colorsCollectionView.selectItem(at: IndexPath(row: selectedColorIndex, section: 0), animated: false, scrollPosition: [])
+            colorsCollectionView.selectItem(
+                at: IndexPath(row: selectedColorIndex,
+                section: 0),
+                animated: false,
+                scrollPosition: []
+            )
```
Comment (David-Henner) [inline wire-ios/Wire-iOS/Sources/UserInterface/SketchColorPickerController.swift]:
```suggestion
            colorsCollectionView.selectItem(
                at: IndexPath(row: selectedColorIndex, section: 0),
                animated: false,
                scrollPosition: []
            )
```

File: wire-ios/Wire-iOS/Sources/UserInterface/SketchColorPickerController.swift [hunk 7]
```diff
@@ -59,14 +64,19 @@ final class SketchColorPickerController: UIViewController {
...
         didSet {
-            colorsCollectionView.selectItem(at: IndexPath(row: selectedColorIndex, section: 0), animated: false, scrollPosition: [])
-
-            delegate?.sketchColorPickerController(self, changedSelectedColor: selectedColor)
+            colorsCollectionView.selectItem(
+                at: IndexPath(row: selectedColorIndex,
+                section: 0),
+                animated: false,
+                scrollPosition: []
+            )
```
Comment (David-Henner) [inline wire-ios/Wire-iOS/Sources/UserInterface/SketchColorPickerController.swift]:
```suggestion
            colorsCollectionView.selectItem(
                at: IndexPath(row: selectedColorIndex, section: 0),
                animated: false,
                scrollPosition: []
            )
```

**Review by David-Henner — APPROVED** [review]

**Pull request merged**